### ISSUE TITLE
Bomberman special crashing

### ISSUE THREAD

**Issue by Xukkorz** [spiritualized1997/openFPGA-SG1000#1]
Game appears to crash when hitting start. Crashed on title as well as in gameplay.

**Comment by GreatHierophant** [comment]
This unlicensed SG-1000 game, like many others requires support for the DahJee RAM extension adapter.

**Comment by Xukkorz** [comment]
Fair enough, I just wanted to mention it because at least one of the versions of this game was able to get into the game on Mega Sg JB. I guess I need to do more testing.
